runtime/core/portable_type/c10/c10/targets.bzl

```diff
@@ -78,9 +78,8 @@ def define_common_targets():
         ] + get_sleef_preprocessor_flags(),
         xplat_exported_deps = [
             "//xplat/caffe2:aten_header",
-            "//xplat/caffe2:generated_aten_config_header",
             "//xplat/caffe2/c10:c10_headers",
-        ],
+        ] + ["//xplat/caffe2:ovrsource_aten_Config.h" if is_arvr_mode() else "//xplat/caffe2:generated_aten_config_header",],
```

Comment:
do we also need to cover line 63 above?

Reply:
shouldn't be needed as arvr only uses xplat_exported_deps and not fbcode_exported_deps